crates/viewer/re_edit_ui/src/lib.rs

```diff
@@ -38,6 +38,7 @@ use re_viewer_context::gpu_bridge::colormap_edit_or_view_ui;
 /// This crate is meant to be a leaf crate in the viewer ecosystem and should only be used by the `re_viewer` crate itself.
 pub fn register_editors(registry: &mut re_viewer_context::ComponentUiRegistry) {
     registry.add_singleline_edit_or_view::<Color>(color::edit_rgba32);
+    registry.add_singleline_edit_or_view::<SolidColor>(color::edit_rgba32);
```

Comment:
Both these colors are no longer rgb only now but rgb with alpha 0 or 1 (until we finally sort out actual transparency :sigh:). We need an adjusted new edit & display ui for this, otherwise we can't display the "not shown" state and a user editing the color can't go back.

![image](https://github.com/user-attachments/assets/ca4ac985-b914-443f-a5b8-a3d4ecbfbbf5)


(can go into a follow-up PR but I consider it ship-blocking for the next release)

Reply:
Yeah, that's a serious problem. (And I noticed it but forgot to mention it in the PR description, whoops.) I think I'll take a stab at making the swatch (and possibly the color picker) handle this, and file an issue to make sure it's tracked.

Reply:
This is moot because we decided to switch away from introducing another component to `FillMode`, so there are no transparent colors used on purpose, any more.